Osiris/Hooks.cpp

```diff
@@ -143,6 +144,16 @@ static HRESULT __stdcall reset(IDirect3DDevice9* device, D3DPRESENT_PARAMETERS*
 
 #endif
 
+static bool __fastcall sendNetMsg(void* networkchannel, void* edx, NetworkMessage& msg, bool bForceReliable, bool bVoice)
+{
+    auto original = hooks->networkChannel.getOriginal<bool, NetworkMessage&, bool, bool>(40, msg, bForceReliable, bVoice);
```

Comment:
`auto original = hooks->networkChannel.getOriginal<bool, 40, NetworkMessage&, bool, bool>(msg, bForceReliable, bVoice);`
lmao did you even test this

Reply:
yeah whats up, it worked. dont want to use it? fine i dont care, its up to you

Reply:
> `auto original = hooks->networkChannel.getOriginal<bool, 40, NetworkMessage&, bool, bool>(msg, bForceReliable, bVoice);`
> lmao did you even test this

yeah whats up, it worked. dont want to use it? fine i dont care, its up to you

Reply:
if you dont want to usse this pull, then fuck off will ya?